exercises/C-variables/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `greeting`
-
+var greeting = "Hello world";
```

Comment:
try to always use `const` to declare variables; `let` can be used too but only use if if the variable _needs_ redefining at some point (normally, this won't be the case, which is why we prefer `const`).

`var` is now outdated, so try to avoid using it :P